src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts

```diff
@@ -58,6 +57,7 @@ describe('HmctsGlobalHeaderComponent', () => {
       declarations: [ HmctsGlobalHeaderComponent ],
       schemas: [ CUSTOM_ELEMENTS_SCHEMA ],
       imports: [
+        HttpClientModule,
```

Comment:
@johnbenjamin-hmcts - as above, please change this to `HttpClientTestingModule`.